src/Dropdown/ButtonTrigger.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+export default class ButtonTrigger extends Component {
+  constructor() {
+    super(...arguments);
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    this.props.onAction(event);
+  }
+
+  getSelectedHtml(item) {
```

Comment:
🎨  Do we really need this method to get the respective value? I know that we're having  similar methods in almost all of our components but we should only add them if doing the operation inline, creating a util or "sub" component is not an option.